packages/replay/src/bin.ts

```diff
@@ -153,6 +161,25 @@ async function commandUploadRecording(id: string, opts: CommandLineOptions) {
   process.exit(recordingId ? 0 : 1);
 }
 
+async function commandLaunchBrowser(browser: string | undefined, opts: CommandLineOptions) {
+  if (!browser || (browser !== "chromium" && browser !== "gecko")) {
+    console.log("Please specify a browser to launch: chromium or gecko");
+    process.exit(1);
+  }
+
+  const browserName = browser as BrowserName;
+  await ensurePlaywrightBrowsersInstalled(browserName);
+
+  const execPath = getPlaywrightBrowserPath(browserName);
+  if (!execPath) {
+    console.log("Path could not be found");
+    process.exit(1);
+  }
+
+  spawn(execPath);
```

Comment:
You should use some launch args here to prevent the welcome screen and maybe some other config. https://peter.sh/experiments/chromium-command-line-switches/#no-first-run